pkg/cluster/config/types.go

```diff
@@ -82,3 +84,13 @@ const (
 	// WorkerRole identifies a node that hosts a Kubernetes worker
 	WorkerRole NodeRole = "worker"
 )
+
+// Network defines networking cluster wide parameters
+type Network struct {
+	// range of IP addresses for the pod network
+	PodSubnet string
+	// range of IP address for service VIPs
+	ServiceSubnet string
+	// true if is IPv6
+	IPv6 bool
```

Comment:
only ipv6? what about dual stack? :thinking:

Reply:
yeah, I think that this needs better planning, let me try to create a google doc to discuss this implementation